deep-rebuild

```diff
@@ -3,7 +3,7 @@
 set -eu
 
 export SILVER_HOME=$(pwd)
-JVM_ARGS="-Xss16M -Xmx4G -jar ../jars/silver.compiler.composed.Default.jar"
+JVM_ARGS="-Xss16M -Xmx6G -jar ../jars/silver.compiler.composed.Default.jar"
```

Comment:
Did this change actually make Silver's memory usage that much worse?  

If so, is this an issue of the Silver compiler being less efficient when compiling pattern matching, or are downstream projects built from the new version of Silver (e.g. ableC) more memory hungry now too?

Reply:
This was left over from when I was trying to fix memory errors due to the bug you fixed with type unification for possibly-decorated nonterminals.  I am reverting this change.